06-working-w-copilot/setting-up-copilot.md

```diff
@@ -0,0 +1,84 @@
+# Setting Up Github's Copilot
+
+## Goals
+We explored how LLM tools like ChatGPT can assist us with code, but what if we had an AI tool built specifically for coding that lives in our IDE where we work? In this lesson we'll get experience with Github's Copilot – one of several AI Assistant tools created to help developers write code quicker by providing suggestions inside our developer environments. 
+
+Our goals for this lesson are to:
+- Understand why we may want to use AI code assistants
+- Get access to Copilot through the GitHub student developer pack
+- Set up the Copilot VS Code extension
+
+## Why use tools like Copilot?
+
+As coding assistants like Copilot become more available and implemented in the industry, it can be helpful to have some experience with how to work with them and where they can be of the most use. But aside from "Because other folks are using them", why do we want to get familiar with these tools? 
+
+The response from many developers is that these tools free you up to spend more time on the unique or complicated parts of your project. We can spend less time looking up boilerplate code online or in a previous project if we can quickly describe what we need and have Copilot generate the code. If we're diving into a new code base, Copilot can help explain sections of code to us so we can get running and ready to contribute faster. 
+
+Like all AI tools, Copilot isn't infallible. When writing code, we still need to know the project and language well enough to understand what functions we'd like to write and how those functions should look. We need that knowledge base to consistently and effectively describe what we want in enough detail to get suggestions, then understand if the suggested code meets our requirements. 
+
+## Accessing Copilot
+
+Github's Copilot is a paid service that is free for students for a limited time through the Github Student Developer Pack. When we redeemed the student developer pack for tools that we used in Unit 2, we also got access to Copilot, but we need to turn on the service. To do so:
+
+1. Check if your student developer pack is still active from Unit 2.
+   - Look for a "PRO" status under "Highlights" in your profile on GitHub (In the upper-right corner of any page, click your profile photo, then click "Your Profile").  
```

Comment:
- minimize references to other parts of the curriculum if they can be avoided
- consistent casing on Student Developer Pack
- the parenthetical is long enough that it feels fine to me as a sentence
```suggestion
Github's Copilot is a paid service that is free for students for a limited time through the Github Student Developer Pack. When redeeming the Student Developer Pack for any of the other benefits it provides, we also gain access to Copilot, but we need to turn on the service. To do so:

1. Check if your Student Developer Pack is currently active.
   - Look for a "PRO" status under "Highlights" in your profile on GitHub. In the upper-right corner of any page, click your profile photo, then click "Your Profile".  
```